### PR TITLE
added better responsive image

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
image

### CODE REVIEW

File: Components/Image/image.html [hunk 1]
```diff
@@ -49,8 +49,8 @@ <h1 class="section-header">Image</h1>
             <h3 class="section-sub-header">Responsive Image</h3>
             <p>This is used to display any image to the 100% of the parent container keeping the
                  aspect ratio in place. Use the classname responsive-image to achieve this in your project.</p>
-            <div class="flex-row">
-                <img src="../../Assets/Images/nature.jpg" alt="Nature"> 
+            <div class="flex-row image-width ">
+                <img class="responsive-image" src="../../Assets/Images/nature.jpg" alt="Nature">
```
Comment (TheAmlanSaikia) [inline Components/Image/image.html:53]: You  can add loading="lazy" in image tag.

File: Components/Image/image.css [hunk 2]
```diff
@@ -1,10 +1,13 @@
 
 .responsive-image {
-    min-width: 100%;
```
Comment (TheAmlanSaikia) [inline Components/Image/image.css:3]: For responsive image you have to add max-inline size, block size , object-fit properties. For more reference refer to Vikram CSS Basic class 3.

**Review by TheAmlanSaikia — APPROVED** [review]

File: Components/Image/image.html (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Image/image.html:53]: sure.

File: Components/Image/image.css (hunk 2 above)
Comment (govardhanreddyc79) [reply Components/Image/image.css:3]: okay

**Review by hrishikeshmane — APPROVED** [review]
Looks good

**Pull request merged**